core/execution/market.go

```diff
@@ -721,6 +721,7 @@ func (m *Market) OnTick(ctx context.Context, t time.Time) bool {
 	timer.EngineTimeCounterAdd()
 
 	m.updateMarketValueProxy()
+	m.updateLiquidityFee(ctx)
```

Comment:
This could be done more efficiently (the most significant part of this computation is calculating the target stake which is done on every tick anyway to emit the market data event so it could either be done there or cached somehow, but either of these options seemed like a premature optimisation. Happy to go ahead with it now if others think it's worth it and justifies additional complexity in the codebase.